Libraries/Animated/src/Interpolation.js

```diff
@@ -167,13 +167,17 @@ function interpolate(
 function colorToRgba(
   input: string
 ): string {
-  var color = tinycolor(input);
-  if (color.isValid()) {
-    var {r, g, b, a} = color.toRgb();
-    return `rgba(${r}, ${g}, ${b}, ${a === undefined ? 1 : a})`;
-  } else {
+  var int32Color = tinycolor(input);
+  if (int32Color === null) {
     return input;
   }
+
+  var a = ((int32Color & 0xff000000) >>> 24) / 255;
```

Comment:
I added

`/* eslint no-bitwise: 0 */`

but github is stupid and doesn't hide comments from older revisions if you didn't change the specific line